open-apparel-registry/open-apparel-registry
# Rebuild the initial migration

## Overview

Two previous changes to the models were not fully captured in the initial migration.

- Renaming Organization to Contributor.
- Adding additional processing statuses to FacilityListItem.

Connects #306

## Notes

This commit was created by deleting the initial migration and running `./scripts/manage makemigrations`

## Testing Instructions

* Run `./scripts/manage resetdb`
* Run `./scripts/manage migrate` and verify that it exits successfully.
* Verify tests pass
